Examples/Detectors/TGeoDetector/include/ActsExamples/TGeoDetector/JsonTGeoDetectorConfig.hpp

```diff
@@ -10,23 +10,26 @@
 
 #include "Acts/Plugins/Json/ActsJson.hpp"
 #include "Acts/Plugins/TGeo/TGeoCylinderDiscSplitter.hpp"
+#include "Acts/Plugins/TGeo/TGeoITkModuleSplitter.hpp"
```

Comment:
Is this a specific version that only works for the `ITk` then we should call it such (and it might have to sit somewhere else, actually) - if it's only inspired by the ITk but is somewhat generic, we can take the `ITk` naming out of it.